components/Main/NewsletterSignUp/NewsletterSignUp.module.css

```diff
@@ -0,0 +1,17 @@
+.section {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.section * {
+  margin: 0;
```

Comment:
Aren't we already resetting the `margin` in the global styles?

Reply:
Yes. global.css set a list of elements' margin to 0:
```css
/* Remove default margin */
body,
h1,
h2,
h3,
h4,
p,
figure,
blockquote,
dl,
dd {
  margin: 0;
}
```

Sometimes I can't help myself adding a `* {margin: 0; }` because I was worried about some elements missed from this list. Should I worry about it?

Reply:
I am also curious about the benefit of using this list instead of 
```css
* {margin: 0; }
```

Reply:
As far as simply using the universal selector I would say it's about [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#selector_types) and that this selector have no effect on this. Whereas the type selectors do.

I wouldn't worry about any being missed. If at some point we do need to change the user agent (browser) margin on an element we can consider whether we want to add it to the global reset otherwise consider creating a component for it that defines the styles we expect for this element in the context of our site.

This is one of the benefits of using a component model is that we should be able to conceptually (as well as technically) scope the concerns of the component to the component. Meaning we can reliably reuse this across our app with consistent results.

Reply:
I understand and thanks for the link.